From 579cb3b0ef3ce3f3e62a7a10da9c39031811cf06 Mon Sep 17 00:00:00 2001
From: Chris Speck <cgspeck@gmail.com>
Date: Wed, 17 Jun 2020 21:40:04 +1000
Subject: [PATCH] Update Dependencies

---
 CMakeLists.txt | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0f5ebf7..7b60e78 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -523,7 +523,8 @@ IF(EXISTS "${CMAKE_ROOT}/Modules/CPack.cmake")
     #-----------------DEB----------------
     SET( CPACK_DEBIAN_PACKAGE_MAINTAINER "Philip G. Lee <rocketman768@gmail.com>" )
     # NOTE: Use the getdependencies script to get the dependencies!
-    SET( CPACK_DEBIAN_PACKAGE_DEPENDS "libc6 (>= 2.3.6-6~), libc6 (>= 2.1.3), libgcc1 (>= 1:4.1.1), libphonon4 (>= 4:4.2.0), libqt4-dbus (>= 4:4.5.3), libqt4-network (>= 4:4.5.3), libqt4-svg (>= 4:4.5.3), libqtwebkit4 (>= 2.3.4.dfsg-9.1), libqt4-xml (>= 4:4.5.3), libqtcore4 (>= 4:4.6.1), libqtgui4 (>= 4:4.5.3), libstdc++6 (>= 4.1.1), phonon (>= 4:4.5.2)" )
+    SET( CPACK_DEBIAN_PACKAGE_DEPENDS "libc6 (>= 2.3.6-6~), libc6 (>= 2.1.3), libgcc1 (>= 1:4.1.1), libstdc++6 (>= 4.1.1), libqt5dbus5 (>= 5.12.8), libqt5network5 (>= 5.12.8), libqt5multimedia5 (>= 5.12.8), libqt5printsupport5 (>= 5.12.8), libqt5webkit5 (>= 5.212.0), libqt5sql5 (>= 5.12.8), libqt5xml5 (>= 5.12.8), libqt5sql5-sqlite (>= 5.12.8), libqt5xml5 (>= 5.12.8), libqt5core5a (>= 5.12.8), libqt5gui5 (>= 5.12.8),
+ libqt5sql5-psql (>= 5.12.8), libqt5svg5 (>= 5.12.8)" )

     SET( CPACK_DEBIAN_PACKAGE_SECTION "misc" )
     SET( CPACK_DEBIAN_PACKAGE_VERSION "${brewtarget_VERSION_STRING}-1")
--
2.25.1

